pkg/arvo/sys/arvo.hoon

```diff
@@ -291,14 +291,23 @@
   |=(b=beam =*(s scot `path`[(s %p p.b) q.b (s r.b) s.b]))
 ::
 ++  de-beam
+  ~/  %de-beam
   |=  p=path
   ^-  (unit beam)
   ?.  ?=([@ @ @ *] p)  ~
   ?~  who=(slaw %p i.p)  ~
   ?~  des=?~(i.t.p (some %$) (slaw %tas i.t.p))  ~  :: XX +sym ;~(pose low (easy %$))
-  ?~  ved=(slay i.t.t.p)  ~
-  ?.  ?=([%$ case] u.ved)  ~
-  `(unit beam)`[~ [`ship`u.who `desk`u.des `case`p.u.ved] t.t.t.p]
+  ?~  ved=(en-case i.t.t.p)  ~
+  `(unit beam)`[~ [`ship`u.who `desk`u.des `case`u.ved] t.t.t.p]
+::
+++  en-case
+  ~/  %en-case
```

Comment:
Please rename to `+de-case`. `+en-case` would be `|=(=case (scot case))`, if you wanted it.